### DISCUSSION TITLE
Space indentation instead of tabs

### DISCUSSION THREAD

**Discussion by pdewouters** [WordPress-Coding-Standards/WordPress-Coding-Standards#2450] [Q&A]
For context, here is the closed issue I filed: https://github.com/WordPress/WordPress-Coding-Standards/issues/2448

I mostly got my custom phpcs config working with spaces, except for multiline arrays:
```
function update_admin_labels() {
  wp_register_script(
    'update-some-admin-labels-js',
    plugins_url( 'my-plugin/js/update-some-admin-labels.js' ),
    [
      'wp-plugins',
      'wp-edit-post',
      'wp-element',
      'wp-components',
      'wp-i18n',
    ],
    1
  );
}
add_action( 'init', 'pdate_admin_labels' );
```
The linter complains unless I indent with 4 spaces.

I feel like there should be an extra property here:
```
<rule ref="WordPress.Arrays.ArrayIndentation">
  <properties>
    <property name="tabIndent" value="false" />
    <property name="indent" value="2"/>
  </properties>
</rule>
```
Any suggestions?

**Comment by jrfnl** [comment]
Closing as duplicate. Your issue was transferred to a discussion already. See #2449
